cfpb/consumerfinance.gov
# Add a note to the ccdb no-js state

ccdb currently renders an empty page when a `no-js` user visits, see https://github.com/cfpb/consumerfinance.gov/pull/6978#issuecomment-1095417416

This adds a note telling them that is expected and links them back to the data download.
<img width="1501" alt="Screen Shot 2022-04-11 at 4 47 04 PM" src="https://user-images.githubusercontent.com/1558033/162830029-21e8e6ae-8d48-4db2-9b77-80fcab24ee94.png">

Testing:
 - Pull and build styles
 - open a tab and disable js (cmd+opt+j, cmd+shift+p, type Disable javascript)
 - visit http://localhost:8000/data-research/consumer-complaints/search/ and the notification should render, with a link back to the data download.
